apps/challenges/urls.py

```diff
@@ -161,4 +161,9 @@
         views.get_challenge_evaluation_cluster_details,
         name="get_challenge_evaluation_cluster_details",
     ),
+    url(
+        r"^challenge/challenge_host_team/(?P<challenge_host_team_pk>[0-9]+)/validate_challenge_config/$",
```

Comment:
We shouldn't need a challenge host team pk to validate the challenge config. Why are we adding this?

Reply:
I will remove this